nx.json

```diff
@@ -15,23 +15,20 @@
   "workspaceLayout": {
     "libsDir": "packages"
   },
-  "extends": "@nrwl/workspace/presets/core.json",
```

Comment:
@igorwessel , did you use [the migration script ](https://nx.dev/nx-api/nx/documents/migrate)to update nx version?

Reply:
Did you run this one too? 

` nx migrate --run-migrations=migrations.json`

I'm asking because it should update the migrations.json file :eyes:

Reply:
I deleted the file, I didn't see the need to keep it, there are only two branch.

Reply:
It was not necessary to specify the file, as there were no migrations it just created the file, and then it was just a matter of running `npx nx migrate --run-migrations`.